ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -3139,6 +3146,21 @@ interface Ethernet81/2
    lldp tlv transmit ztp vlan 112
    spanning-tree portfast
 !
+interface Ethernet81/3
+   description Traffic Engineering Interface
+   no shutdown
+   no switchport
+   ip address 100.64.127.0/31
+   traffic-engineering
+   traffic-engineering administrative-group 3,testgrp,15-29
+!
+interface Ethernet81/4
+   description Traffic Engineering Interface
+   no shutdown
+   no switchport
+   ip address 100.64.127.0/31
+   traffic-engineering administrative-group 4,testgrp,7-100
```

Comment:
Given that this is what is rendered in config:

```
+++ session:/42-session-config
interface Ethernet3
+   traffic-engineering administrative-group 4,7-100,testgrp
```

I would change the order in the inputs to match - (yes it is still valid like this but user need to do the ordering if they want CLI to match AVD generated config)

Reply:
Done.